### PR TITLE
Build new kubernetes managed page

### LINKED ISSUE #10471: [Rebuild /kubernetes/managed] build page
Design ticket: https://github.com/canonical-web-and-design/ubuntu.com/issues/10470

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Rebuild `/managed/kubernetes`
- Removed the local p-separator file as the vanilla one is the same.

## QA

- View page at: https://ubuntu-com-10550.demos.haus/kubernetes/managed
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the page against the [design](https://github.com/canonical-web-and-design/ubuntu.com/issues/10470#issuecomment-931418091) and [copy doc](https://docs.google.com/document/d/1Qv379nYnod9e4jMrsdDND8D7mbPyqELi2lowjkfD-gU/edit#) and resolve all comments
- Resources section is slightly different from the design as I used a h4 instead of muted heading - suggested [here](https://github.com/canonical-web-and-design/vanilla-framework/pull/3935)

## Issue / Card

Fixes #10471

### CODE REVIEW

**Comment by spencerbygraves** [comment]
@bethcollins92, looking good, a couple of small things please:

- First green button should read "Talk to an expert"
- Looks like the separator in the second section has extra margin, can it be 2rem top and bottom please, same as the "Managed IT services..." section further down

Thanks.

**Comment by bethcollins92** [comment]
Thanks for the review - I've fixed the button text. I'm not too sure about the separators though as I haven't added any extra margin and just used the vanilla `p-separator` class. 

Here is a screen shot from the second section: 
Screenshot 2021-10-05 at 08 37 54

And here is a screenshot from the "Managed IT services" section: 
Screenshot 2021-10-05 at 08 38 15

**Comment by spencerbygraves** [comment]
@bethcollins92 

> I'm not too sure about the separators though as I haven't added any extra margin and just used the vanilla p-separator class.

The `p-separator` class has 2rem top and bottom [on the docs site](https://vanillaframework.io/docs/base/separators#separator)

image

**Comment by pmahnke** [comment]
Spoke with @spencerbygraves, we are leaving this as is and he will look at Vanilla's margins

File: templates/kubernetes/managed.html [hunk 1]
```diff
@@ -37,485 +38,395 @@ <h1>
...
+        Predictable economics
+      </h3>
+      <p>Predictable pricing model that gives you the flexibility to run Kubernetes on VMs, bare-metal or public cloud. Our predictive capacity planning helps you prepare for the future and grow your workloads without any spikes. </p>
     </div>
-    <div class="col-4 p-divider__block">
-      <h2 class="p-heading--three">Reliable</h2>
-      <p>Every Kubernetes cluster built by Canonical is always automatically upgradable to newer versions. Absolutely no snowflakes!</p>
+    <div class="col-4">
+      <h3 class="p-heading--4">
+        Customizable clusters
```
Comment (pmahnke) [inline templates/kubernetes/managed.html:52]:
```suggestion
        Customisable clusters
```

File: templates/kubernetes/managed.html [hunk 2]
```diff
@@ -37,485 +38,395 @@ <h1>
...
+
   <div class="u-fixed-width">
-    <ul class="p-matrix is-trisected">
-      <li class="p-matrix__item">
-        <div class="p-matrix__content">
-          <h3 class="p-matrix__title p-heading--four">Guaranteed SLA</h3>
-          <p class="p-matrix__desc">Our expertise, based on data from hundreds of clouds, underpins your Service Level Agreement. Our operations are certified, with regular independent audits.</p>
+    <div class="p-logo-section">
+      <p class="p-logo-section__title u-align-text--center">
+        Supported
```
Comment (pmahnke) [inline templates/kubernetes/managed.html:259]:
```suggestion
        Supported clouds
```

**Review by pmahnke — APPROVED** [review]
LGTM with 2 updates.

**Pull request merged**